PRESS-RELEASE.md

```diff
@@ -1,44 +1,28 @@
-
-If the benefits listed don't sound very interesting or exciting to customers, then perhaps they're not (and shouldn't be built). Instead, the product manager should keep iterating on the press release until they've come up with benefits that actually sound like benefits. Iterating on a press release is a lot less expensive than iterating on the product itself (and quicker!).
-
-If the press release is more than a page and a half, it is probably too long. Keep it simple. 3-4 sentences for most paragraphs. Cut out the fat. Don't make it into a spec. You can accompany the press release with a FAQ that answers all of the other business or execution questions so the press release can stay focused on what the customer gets. My rule of thumb is that if the press release is hard to write, then the product is probably going to suck. Keep working at it until the outline for each paragraph flows. 
-
-Oh, and I also like to write press-releases in what I call "Oprah-speak" for mainstream consumer products. Imagine you're sitting on Oprah's couch and have just explained the product to her, and then you listen as she explains it to her audience. That's "Oprah-speak", not "Geek-speak".
-
-Once the project moves into development, the press release can be used as a touchstone; a guiding light. The product team can ask themselves, "Are we building what is in the press release?" If they find they're spending time building things that aren't in the press release (overbuilding), they need to ask themselves why. This keeps product development focused on achieving the customer benefits and not building extraneous stuff that takes longer to build, takes resources to maintain, and doesn't provide real customer benefit (at least not enough to warrant inclusion in the press release).
- -->
  
 ## LANshark ##
-  <!-- > Name the product in a way the reader (i.e. your target customers) will understand. -->
+ > Working Title: Augmented History Tour
 
 ## Use Augmented Reality to discover facts about your city ##
-  <!-- > Describe who the market for the product is and what benefit they get. One sentence only underneath the title. -->
+  > This app is for history lovers and travelers who want to experience a new way of learning about the places they are in.
 
 ## Summary ##
-  > Allows people to use augmented reality on their mobile device to see points of interest their current area, and access that information by using their camera in real time to create an interactive cultural experience.
+  > Allows people to use augmented reality on their mobile device discover points of interest and facts related their current area, and, to access that information by using their smartPhones camera in real time to create an interactive cultural experience.
```

Comment:
Could you combine these sentences into one? "Allows people to use augmented reality on their mobile device discover points of interest and facts related their current area, and, to access that information by using their smartPhones camera in real time to create an interactive cultural experience." instead of this, maybe something like this "Allows people to use augmented reality on their smartphone's camera in real time to discover points of interest and facts related their current area to create an interactive cultural experience. "